carma_wm_ctrl/test/WMBroadcasterTest.cpp

```diff
@@ -815,21 +822,57 @@ TEST(WMBroadcaster, geofenceFromMsgTest)
   msg_v01.geometry.nodes.push_back(pt);
   msg_v01.params_exists = true;
   
-  // test maxspeed
+  // test maxspeed - config limit inactive
   msg_v01.params.detail.choice = cav_msgs::TrafficControlDetail::MAXSPEED_CHOICE;
   msg_v01.params.detail.maxspeed = 50;
   auto gf_ptr = wmb.geofenceFromMsg(msg_v01);
   ASSERT_TRUE(gf_ptr->regulatory_element_->attribute(lanelet::AttributeName::Subtype).value().compare(lanelet::DigitalSpeedLimit::RuleName) == 0);
   lanelet::DigitalSpeedLimitPtr max_speed = std::dynamic_pointer_cast<lanelet::DigitalSpeedLimit>(gf_ptr->regulatory_element_);
   ASSERT_NEAR(max_speed->speed_limit_.value(), 22.352, 0.00001);
+  ASSERT_LT(max_speed->speed_limit_, 80_mph);//Check that the maximum speed limit is not larger than 80_mph
+  ROS_WARN_STREAM("Maximum speed limit is valid.");
```

Comment:
let's remove any print statement in the tests that you as we know if it passed or not from the tests themselves. Except clarifying ERROR msgs that the code generate while testing that they are expected